url/query/params.go

```diff
@@ -0,0 +1,53 @@
+package query
+
+import (
+	"net/url"
+	"strings"
+)
+
+// Operators for operator -> values
+// empty operator means that no operator is found near the value; default one should be applied
```

Comment:
Default one is?

Reply:
Default one used by server. For example one could say that if no operator asc or desc in sort parameter, then server will apply as default asc sorting. But this is an implementation detail and kind od agreement that team can come what they need. Maybe this statement should not be stated here...

Reply:
Ah, I see. Maybe we should be clear here that the service decides the default.